crates/bevy_transform/src/components/children.rs

```diff
@@ -29,6 +31,33 @@ impl Children {
     pub fn swap(&mut self, a_index: usize, b_index: usize) {
         self.0.swap(a_index, b_index);
     }
+
+    /// Returns result of the subquery filtered by the children
+    ///
+    /// Example:
+    /// ```
+    /// fn example_system(
+    ///    children: Query<&Children, With<Sprite>>,
+    ///    transforms: Query<&Transform, With<Sprite>>,
+    /// ) {
+    ///     for transform in children.subquery(transforms) {
```

Comment:
Wouldn't `children` here be a `Query` and not the `Children` component?

Reply:
Oh, yep. That example needs an extra iteration. Good catch